confignode/src/main/java/org/apache/iotdb/confignode/procedure/impl/pipe/task/DropPipeProcedureV2.java

```diff
@@ -43,6 +44,8 @@ public class DropPipeProcedureV2 extends AbstractOperatePipeProcedureV2 {
 
   private String pipeName;
 
+  private PipeMeta pipeMeta;
```

Comment:
Remove it. PipeMeta should NOT be directly accessed here.

Reply:
Then how to store the removed PipeMeta here? Can we leave it undeleted until the rpc to the datanode is  done？